demos/showcase/src/App.js

```diff
@@ -216,7 +216,7 @@ export default () => {
             }
         >
             <div style={{ height: '100%', display: 'flex', flexDirection: 'column' }}>
-                <AppBar position={'static'} color={'primary'}>
+                <AppBar position={'sticky'} color={'primary'}>
```

Comment:
Can you add routing to the showcase so we can see the effects of the scroll behavior?

Reply:
Yep added this to the demo.